bin/sender.py

```diff
@@ -62,40 +62,102 @@ def main():
         sys.exit(1)
     
     log = logging.getLogger('ssmsend')
-    
+
+    # Set defaults for MQ-BROKER only variables
+    use_ssl = None
+    # Set defaults for AMS only variables
+    token = ""
```

Comment:
Look at defaulting this in init (currently goes into password)

Reply:
Note that empty string is needed as AMS only uses that to define abscence of token.